platform/nodejs/eslintrc.json

```diff
@@ -9,7 +9,21 @@
         "sourceType": "module"
     },
     "rules": {
-        "indent": [ "warn", 4 ],
+        "indent": [
+          "warn",
+          4,
+          {
+            "ArrayExpression": "off",
+            "CallExpression": { "arguments": "first" },
+            "MemberExpression": "off",
+            "ObjectExpression": "off",
+            "ignoreComments": true,
+            "ignoredNodes": [
+              "AssignmentExpression:has(Literal)",
+              "LogicalExpression"
```

Comment:
Now I've added an exception for logical expressions.

This removes the warnings for code like this:

```js
        return this.optionUnitBits === this.DOMAIN_BIT &&
            this.isRegex === false && (
                this.pattern === '*' || (
                    this.anchor === 0b010 &&
                    /^(?:http[s*]?:(?:\/\/)?)$/.test(this.pattern)
                )
            ) &&
            this.domainOpt.indexOf('~') === -1;
```

It may be too broad an exception so I'll try to narrow it down later.